scripts/yul_coverage.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+set -e
+
+ROOT_DIR="$(dirname "$0")"/..
+
+for arg in "$@"; do
+  if [ "$arg" == "--no-stats" ]; then
+    NO_STATS=1
+  elif [ "$arg" == "--no-errors" ]; then
+    SHOW_NO_ERRORS=1
```

Comment:
I think this name is a bit misleading. At first I thought it would suppress errors. Maybe something like `--other-output` or `--accepted-lines` would be better?

Reply:
I changed this now to `--successful`. It should return all successful tests.